hyperspy/model.py

```diff
@@ -1480,7 +1483,11 @@ def print_current_values(self, only_free=True):
                 parameters = component.free_parameters if only_free \
                     else component.parameters
                 for parameter in parameters:
-                    if not hasattr(parameter.value, '__iter__'):
+                    if hasattr(parameter.value, '__iter__'):
+                        if skip_multi:
+                            for idx in range(len(parameter.value)):
+                                print("\t\ta[%d]\t%g" % (idx, parameter.value[idx]))
+                    else:
```

Comment:
Also, wouldn't it be better to print the parameter name instead of ``a``? This is what I get when using ``skip_multi=True`` (should be ``False``) on an EELS model:

```python


Components	Parameter	Value
PowerLaw
		A	40
		r	-0
Fe_M3
		a[0]	0
		a[1]	0
		a[2]	0
		a[3]	0
		a[4]	0
		a[5]	0
		a[6]	0
		a[7]	0
		a[8]	0
		a[9]	0
		a[10]	0
		a[11]	0
		a[12]	0
		a[13]	0
		a[14]	0
		a[15]	0
		a[16]	0
		a[17]	0
		a[18]	0
		a[19]	0
		a[20]	0
		a[21]	0
		a[22]	0
		a[23]	0
		a[24]	0
		intensity	1
```